latest-langs

```diff
@@ -13,6 +13,7 @@ constant %paths = (
     'Assembly'     => 'registry.npmjs.org/@defasm/core/latest',
     'Bash'         => 'en.wikipedia.org/wiki/Bash_(Unix_shell)',
     'BASIC'        => 'en.wikipedia.org/wiki/FreeBASIC',
+    'Befunge'      => 'github.com/VorpalBlade/cfunge',
```

Comment:
```bash
oh@Yewzir:~/code-golf$ ./latest-langs Befunge
     Befunge     b7bf640 → b7bf640
```
The number-of-commits-versioning caused the execution process to abruptly terminate when it reached this particular entry.